### PR TITLE
EC3 widget

### PR DESCRIPTION
**Pull request by Simonas-A**

### CODE REVIEW

**Comment by aruniverse** [comment]
@Simonas-A , whoever you are, please ping me on teams.

File: rush.json [hunk 1]
```diff
@@ -66,6 +66,11 @@
       "packageName": "@itwin/infrastructure-iot-react",
       "projectFolder": "packages/itwin/infrastructure-iot",
       "versionPolicyName": "infrastructure-iot"
+    },
+    {
+      "packageName": "@itwin/ec3-react",
```
Comment (MindaugasButkus) [inline rush.json:71]: I think the project name should be `@itwin/ec3-widget-react`.

File: common/config/rush/pnpm-lock.yaml [hunk 2]
```diff
@@ -5,742 +5,7 @@ importers:
   .:
     specifiers: {}
```
Comment (MindaugasButkus) [inline common/config/rush/pnpm-lock.yaml:7]: It is very suspicious that there are that many lines removed and that little lines added to the lock file. Could you pull this repo to a fresh directory, revert the changes in pnpm-lock.yaml (download the file from master) and run `rush install`?

File: packages/itwin/ec3-widget/package.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@itwin/ec3-react",
+  "version": "1.0.0",
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/package.json:3]: Downgrade version to 0.1.0 for initial (pre)release.

File: packages/itwin/ec3-widget/package.json [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  "description": "",
+  "main": "lib/cjs/e-c-3-widget.js",
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/package.json:5]: There should be no dashes in EC3. `"main": "lib/cjs/ec3-widget.js"

File: packages/itwin/ec3-widget/package.json [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    "cpx2": "^3.0.0",
+    "eslint": "^7.32.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-react": "^7.25.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "jest": "^27.5.1",
+    "jest-cli": "^27.5.1",
+    "jest-transform-stub": "^2.0.0",
+    "msw": "^0.39.2",
+    "react": "^18.2.0",
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/package.json:98]: Downgrade to what is required from peerDependencies. This will also make it consistent with other packages.

File: packages/itwin/ec3-widget/package.json [hunk 6]
```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@itwin/ec3-react",
+  "version": "1.0.0",
+  "description": "",
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/package.json:4]: Description is missing.

File: packages/itwin/ec3-widget/package.json [hunk 7]
```diff
@@ -0,0 +1,114 @@
+{
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/package.json:1]: Keywords are missing. Please grab them from the OneClickLCA widget.

File: packages/itwin/ec3-widget/react-app-env.d.ts [hunk 8]
```diff
@@ -0,0 +1,5 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/// <reference types="@bentley/react-scripts" />
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/react-app-env.d.ts:5]: I don't think this file should be here.

File: packages/itwin/ec3-widget/src/components/EC3/generated/api.ts [hunk 9]
```diff
@@ -0,0 +1,712 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/EC3/generated/api.ts:1]: Do not add a generated client. I guess it would be better to write a non-generated client here that could be copied over to the insights-client repo later. This generated client adds a lot of information and complexity that is not really needed here.

File: packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+    const baseUrl = prefixUrl(CC_BASE_PATH, process.env.IMJS_URL_PREFIX);
+    this._oclcaApi = new OneClickLCAApi(undefined, baseUrl); // TODO: change to baseUrl in deployment
+  }
+
+  public async getEC3AccessToken(username: string, password: string) {
+    if (username === undefined || password === undefined) {
+      return undefined;
+    }
+
+    //const mode: RequestMode = "no-cors";
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts:30]: Delete all code that is commented out.

File: packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+const prefixUrl = (baseUrl?: string, prefix?: string) => {
+  if (prefix && baseUrl) {
+    return baseUrl.replace("api.bentley.com", `${prefix}api.bentley.com`);
+  }
+  return baseUrl;
+};
+
+export default class EC3Client {
+  private _oclcaApi: OneClickLCAApi;
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts:19]: Do not use OneClickLCA APIs for EC3.

File: packages/itwin/ec3-widget/src/components/EC3.tsx [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import Templates from "./Templates";
+import "./Templates.scss";
+
+const EC3 = () => {
+  return (
+    <div className="e_c_3-container">
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/EC3.tsx:11]: `className="ec3-container"`.

File: packages/itwin/ec3-widget/src/components/templateClient.tsx [hunk 13]
```diff
@@ -0,0 +1,121 @@
...
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Guid } from "@itwin/core-bentley";
+import { Template, Label, Material } from "./Template"
+
+// For now only one template for each report.
+
+export default class TemplateClient {
+  getTemplatesT(): Template[] {
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/templateClient.tsx:11]: If the method is public - mark it as `public`.

File: packages/itwin/ec3-widget/src/components/templateClient.tsx [hunk 14]
```diff
@@ -0,0 +1,121 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Guid } from "@itwin/core-bentley";
+import { Template, Label, Material } from "./Template"
+
+// For now only one template for each report.
+
+export default class TemplateClient {
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/templateClient.tsx:10]: File name does not match the class name. Rename the file (make the `t` upper case).

File: packages/itwin/ec3-widget/src/components/templateClient.tsx [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+
+    //templates[0] =
+    //const text = localStorage.getItem("template");
+
+    //return JSON.parse(text ?? "");
+  }
+
+
+  getTemplateT(templateId: string): Template | undefined {
+
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/templateClient.tsx:37]: No trailing or leading empty lines in methods and no double empty lines.

File: packages/itwin/ec3-widget/src/hooks/useValidator.ts [hunk 16]
```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { useState } from "react";
+import SimpleReactValidator from "simple-react-validator";
+
+export const Validators = {
+  NoDuplicateUnderscore: {
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/hooks/useValidator.ts:9]: Do these need to start with an uppercase letter?

File: packages/itwin/ec3-widget/src/test/Templates.test.tsx [hunk 17]
```diff
@@ -0,0 +1,265 @@
...
...EntityType Name=\"Demo_Slabs_Mapping_bd2cc9f9ab5c42e6bc1757b7b7ca5bad\"><Property Name=\"ECInstanceId\" Type=\"Edm.String\" /><Property Name=\"ECClassId\" Type=\"Edm.String\" /><Property Name=\"UserLabel\" Type=\"Edm.String\" /><Property Name=\"BBoxLow\" Type=\"Edm.String\" /><Property Name=\"BBoxHigh\" Type=\"Edm.String\" /><Property Name=\"volume\" Type=\"Edm.Double\" /></EntityType></Schema><Schema Namespace=\"Default\" xmlns=\"http://docs.oasis-open.org/odata/ns/edm\"><EntityContainer Name=\"Container\"><EntitySet Name=\"mapping0\" EntityType=\"Insights_and_Reporting_Extractor.mapping0\" /><EntitySet Name=\"Demo_RailColumn_Mapping_bd2cc9f9ab5c42e6bc1757b7b7ca5bad\" EntityType=\"Insights_and_Reporting_Extractor.Demo_RailColumn_Mapping_bd2cc9f9ab5c42e6bc1757b7b7ca5bad\" /><EntitySet Name=\"Demo_RoofBeam_Mapping_bd2cc9f9ab5c42e6bc1757b7b7ca5bad\" EntityType=\"Insights_and_Reporting_Extractor.Demo_RoofBeam_Mapping_bd2cc9f9ab5c42e6bc1757b7b7ca5bad\" /><EntitySet Name=\"Demo_Slabs_Mapping_bd2cc9f9ab5c42e6bc1757b7b7ca5bad\" EntityType=\"Insights_and_Reporting_Extractor.Demo_Slabs_Mapping_bd2cc9f9ab5c42e6bc1757b7b7ca5bad\" /></EntityContainer></Schema></edmx:DataServices></edmx:Edmx>";
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/test/Templates.test.tsx:92]: Move test data to a separate file where it could be formatted properly.

File: packages/itwin/ec3-widget/src/test/Templates.test.tsx [hunk 18]
```diff
@@ -0,0 +1,265 @@
...
+import { useActiveIModelConnection } from "@itwin/appui-react";
+import type { ReportCollection } from "@itwin/insights-client";
+import faker from "@faker-js/faker";
+import {  LabeledSelect } from "@itwin/itwinui-react";
+import { setupServer } from "msw/node";
+import { rest } from "msw";
+import userEvent from "@testing-library/user-event";
+//import type { ODataResponse } from "./generated/api";
+
+afterAll(() => {
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/test/Templates.test.tsx:24]: Move all of the required setup into the `describe` function. Usually there should be very little code (mainly imports) outside of `describe` in test files.

File: packages/itwin/ec3-widget/tsconfig.json [hunk 19]
```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
+  "compilerOptions": {
+    "skipLibCheck": true,
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/tsconfig.json:4]: `skipLibCheck` and `jsx` match what is in the `tsconfig-base.json` file, to they can be removed here. Could you explain why you need `preserveSymlinks` and `esModuleInterop`?

File: packages/itwin/ec3-widget/package.json (hunk 3 above)
Comment (Guillar1) [reply packages/itwin/ec3-widget/package.json:3]: I believe it should just be 0.0.0 and let the pipeline bump it for us with `rush change`

**Comment by Guillar1** [comment]
Missing `README.md`, `LICENSE.md`, and you need to add the widget to the root `README.md`

File: packages/itwin/ec3-widget/src/e-c-3-widget.ts [hunk 20]
```diff
@@ -0,0 +1,6 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/src/e-c-3-widget.ts:1]: ec3-widget-react.ts

File: packages/itwin/ec3-widget/package.json [hunk 21]
```diff
@@ -0,0 +1,114 @@
...
+    }
+  },
+  "dependencies": {
+    "@itwin/imodels-access-frontend": "^1.0.1",
+    "@itwin/imodels-client-management": "^1.0.1",
+    "@itwin/insights-client": "^0.2.0",
+    "@itwin/itwinui-icons-color-react": "^1.0.1",
+    "@itwin/itwinui-icons-react": "^1.7.0",
+    "@itwin/itwinui-react": "~1.42.0",
+    "react-redux": "^8.0.2",
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/package.json:52]: Not being used.

File: packages/itwin/ec3-widget/package.json [hunk 22]
```diff
@@ -0,0 +1,114 @@
...
+    ],
+    "extends": "plugin:@itwin/ui",
+    "rules": {
+      "no-duplicate-imports": "off",
+      "@typescript-eslint/consistent-type-imports": "error",
+      "deprecation/deprecation": "off"
+    }
+  },
+  "dependencies": {
+    "@itwin/imodels-access-frontend": "^1.0.1",
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/package.json:46]: A lot of these packages in `dependencies` are not being used.

File: packages/itwin/ec3-widget/package.json [hunk 23]
```diff
@@ -0,0 +1,110 @@
...
+      "no-duplicate-imports": "off",
+      "@typescript-eslint/consistent-type-imports": "error",
+      "deprecation/deprecation": "off"
+    }
+  },
+  "dependencies": {
+    "@itwin/insights-client": "^0.2.0",
+    "@itwin/itwinui-icons-react": "^1.7.0",
+    "@itwin/itwinui-react": "~1.42.0",
+    "react-redux": "^7.2.2",
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/package.json:56]: This should be a dev dependency as it is a peer of appui-react.

File: packages/itwin/ec3-widget/package.json (hunk 7 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/package.json:1]: Keywords have been added

File: rush.json (hunk 1 above)
Comment (Simonas-A) [reply rush.json]: project name has been changed

File: packages/itwin/ec3-widget/package.json (hunk 6 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/package.json]: Description has been added

File: packages/itwin/ec3-widget/package.json (hunk 4 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/package.json]: Dashes have been removed

File: packages/itwin/ec3-widget/react-app-env.d.ts (hunk 8 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/react-app-env.d.ts]: File has been deleted

File: packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts (hunk 11 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts]: Unnecessary code has been removed

File: packages/itwin/ec3-widget/src/components/templateClient.tsx (hunk 14 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/templateClient.tsx]: File has been renamed

File: packages/itwin/ec3-widget/src/components/templateClient.tsx (hunk 13 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/templateClient.tsx]: public keyword added, unnecessary methods removed

File: packages/itwin/ec3-widget/src/components/templateClient.tsx (hunk 15 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/templateClient.tsx]: Unnecessary empty lines have been removed

File: packages/itwin/ec3-widget/package.json (hunk 3 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/package.json]: Changed the version to 0.0.0

File: packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts (hunk 10 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/EC3/EC3Client.ts]: Commented out code has been deleted

File: packages/itwin/ec3-widget/src/components/TemplateClient.tsx [hunk 24]
```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { Guid } from "@itwin/core-bentley";
+import { Template, Label, Material } from "./Template"
+
+// For now only one template for each report.
+
+export default class TemplateClient {
```
Comment (MindaugasButkus) [inline packages/itwin/ec3-widget/src/components/TemplateClient.tsx:10]: This should not be a `tsx` file.

File: packages/itwin/ec3-widget/src/components/EC3/generated/api.ts (hunk 9 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/EC3/generated/api.ts]: generated folder was redundant and the files were removed

File: packages/itwin/ec3-widget/src/hooks/useValidator.ts (hunk 16 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/hooks/useValidator.ts]: Unnecessary validators were removed. Only required validator is left.

File: packages/itwin/ec3-widget/src/test/Templates.test.tsx (hunk 17 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/test/Templates.test.tsx]: Test data was moved to a different file

File: packages/itwin/ec3-widget/src/test/Templates.test.tsx (hunk 18 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/test/Templates.test.tsx]: Most of the setups has been moved to describe function

File: packages/itwin/ec3-widget/src/e-c-3-widget.ts (hunk 20 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/e-c-3-widget.ts]: File was renamed to ec3-widget-react.ts

File: packages/itwin/ec3-widget/package.json (hunk 22 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/package.json]: Unnecessary packages were removed

File: packages/itwin/ec3-widget/src/components/ExportModal.scss [hunk 25]
```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.e_c_3 {
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/src/components/ExportModal.scss:7]: `.ec3`

File: packages/itwin/ec3-widget/src/components/GroupSelector.scss [hunk 26]
```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.e_c_3 {
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/src/components/GroupSelector.scss:7]: `.ec3`

File: packages/itwin/ec3-widget/src/components/HorizontalTile.scss [hunk 27]
```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.rcw-horizontal-tile-container {
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/src/components/HorizontalTile.scss:7]: Should be `.ec3-horizontal-tile-container`

File: packages/itwin/ec3-widget/src/components/LabelAction.scss [hunk 28]
```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import '~@itwin/itwinui-css/scss/variables';
+
+.label-details-container {
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/src/components/LabelAction.scss:7]: Can you add ec3 in front of these selectors.

File: packages/itwin/ec3-widget/src/components/LabelTile.scss [hunk 29]
```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import '~@itwin/itwinui-css/scss/variables';
+
+.gmw-label-tile-container {
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/src/components/LabelTile.scss:7]: `.ec3-label-tile-container`

File: packages/itwin/ec3-widget/src/components/TemplateMenu.scss [hunk 30]
```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.template {
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/src/components/TemplateMenu.scss:7]: Missing a prefix `ec3`

File: packages/itwin/ec3-widget/package.json (hunk 21 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/package.json]: react-redux was removed from dependencies

File: packages/itwin/ec3-widget/src/components/ExportModal.scss (hunk 25 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/ExportModal.scss]: selector renamed

File: packages/itwin/ec3-widget/src/components/GroupSelector.scss (hunk 26 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/GroupSelector.scss]: File was redundant. Deleted.

File: packages/itwin/ec3-widget/src/components/HorizontalTile.scss (hunk 27 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/HorizontalTile.scss]: Selector renamed

File: packages/itwin/ec3-widget/src/components/LabelAction.scss (hunk 28 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/LabelAction.scss]: ec3 added in front of the selectors

File: packages/itwin/ec3-widget/src/components/LabelTile.scss (hunk 29 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/LabelTile.scss]: Selector renamed

File: packages/itwin/ec3-widget/src/components/TemplateMenu.scss (hunk 30 above)
Comment (Simonas-A) [reply packages/itwin/ec3-widget/src/components/TemplateMenu.scss]: Redundant styles removed. Prefix added.

File: packages/itwin/ec3-widget/package.json [hunk 31]
```diff
@@ -0,0 +1,110 @@
...
+    }
+  },
+  "dependencies": {
+    "@itwin/insights-client": "^0.2.0",
+    "@itwin/itwinui-icons-react": "^1.7.0",
+    "@itwin/itwinui-react": "~1.42.0",
+    "simple-react-validator": "^1.6.1"
+  },
+  "devDependencies": {
+    "react-redux": "^7.2.2",
```
Comment (Guillar1) [inline packages/itwin/ec3-widget/package.json:59]: This needs to be sorted.
